spec/models/user_group_spec.rb

```diff
@@ -5,7 +5,7 @@
   let(:named) { user_group }
   let(:activatable) { user_group }
   let(:owner) { user_group }
-  let(:owned) { build(:project, owner: owner) }
+  let(:owned) { create(:project, owner: owner) }
```

Comment:
I couldn't figure out a workarond to get `user_group.owned_resources` to come out as expected in order for the `owner.rb` shared specs to work  other than to create the `owned` resource. 

Any tips in order to avoid create would be great

Reply:
Instead of creating the project for every spec in this file you can instead do it for just the failing 'is an owner' ones to save on the spec cost setup
```
it_behaves_like "is an owner" do
  before { owned.save }
end
```

Reply:
ooh yep that works. thanks!